simplified-main/src/main/java/org/librarysimplified/main/MainFragmentListenerDelegate.kt

```diff
@@ -669,7 +669,14 @@ internal class MainFragmentListenerDelegate(
   }
 
   private fun goUpwards() {
-    this.navigator.popBackStack()
+    logger.error("goUpwards(), stack size: {}", navigator.backStackSize())
```

Comment:
error or debug?

Reply:
Good catch! I sometimes use "error" for the log level just to make some messages show up better when debugging something. :sweat_smile: